modules/widgets/blog-stats.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * Blog Stats Widget.
+ *
+ * @since 4.5.0
+ *
+ * @package Jetpack
+ */
+
+/**
+ * Disable direct access/execution to/of the widget code.
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Blog Stats Widget.
+ *
+ * Displays all time stats for that site.
+ *
+ * @since 4.5.0
+ */
+class Jetpack_Blog_Stats_Widget extends WP_Widget {
+
+	/**
+	 * Constructor
+	 */
+	function __construct() {
+		$widget_ops = array(
+			'classname' => 'blog-stats',
+			'description' => __( 'Show a hit counter for your blog.', 'jetpack' ),
+			'customize_selective_refresh' => true,
+		);
+		parent::__construct(
+			'blog-stats',
+			/** This filter is documented in modules/widgets/facebook-likebox.php */
+			apply_filters( 'jetpack_widget_name', __( 'Blog Stats', 'jetpack' ) ),
+			$widget_ops
+		);
+		$this->alt_option_name = 'widget_statscounter';
```

Comment:
I believe it was used with old widgets; I think this would be useful for folks transferring from WordPress.com to WordPress.org, if they were using the WordPress.com Blog Stats Widget on their old site. The WordPress.com Blog Stats Widget uses the same values.